tilequeue/worker.py

```diff
@@ -161,8 +161,12 @@ def __call__(self, stop):
                 cut_coords.extend(coord_children_range(coord, nominal_zoom))
             max_zoom = 16 - self.metatile_zoom
 
-            assert coord.zoom <= max_zoom, \
-                "Job coordinates above max zoom are not supported"
+            if coord.zoom > max_zoom:
+                self.logger.log(
+                    logging.WARNING,
+                    'Job coordinates above max zoom are not supported, '
+                    'skipping %r > %d' % (coord, max_zoom))
+                continue
```

Comment:
I might have put that in there to signal that there was a disconnect somewhere. The risk with logging warnings is that we might not see it right away though. Maybe once we improve the logging and metrics gathering in tilequeue as well we'll be in a better position to notice these kinds of things.